doc/design/staging-1.md

```diff
@@ -0,0 +1,37 @@
+# Staging Environment and Procedures
+
+## Production Environment
+* Django API server, Postgres, and CloudAMQP running in Heroku
+* Sponsored S3 bucket in the sponsored account in AWS
+* Web GUI served through Netlify
+* Sentry instance
+
+## Staging Environment
+* Django API server, Postgres, and CloudAMQP running in Heroku
+* Private S3 bucket in the project account in AWS
```

Comment:
I intended for this document to be a description of the development > staging > production process, not a comprehensive documentation of our environment. I think adding that information will only make the document expire quicker as we tweak things and forget to update it here.

It would be nice to have that information somewhere, but I don't think it should be here.

Reply:
I agree with @dchiquito on this: let's leave that decision to execution time, and then document somewhere that isn't this design doc.